src/modules/recipes/NewRecipeForm/index.jsx

```diff
@@ -0,0 +1,123 @@
+import { useForm } from "react-hook-form";
+import "./styles.css";
+import { RecipesProvider } from "../RecipesProvider";
+
+export const NewRecipeForm = () => {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+    reset,
+  } = useForm({
+    defaultValues: {
+      strMeal: "",
+      strCategory: "Beef",
+      strInstructions: "",
+      strIngredient: "",
+      strMealThumb: "",
+      cookingTime: "",
+    },
+  });
+  const categories = watch(
```

Comment:
Remove, this is causing an issue with this page since it's not possible to map through the values of this variable. In fact, you don't even use the result of the watch function anywhere.